webrtc.html

```diff
@@ -7762,6 +7778,7 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
              USVString      protocol = "";
              boolean        negotiated = false;
              unsigned short id;
+             RTCPriorityType priority;
```

Comment:
If we add a default value of "low" here, we don't have to special-case the missing attribute in the handling instructions above.

RTCPriorityType priority = "low";

(Note: I can't tell what the default for RTP channels is from the document. "low" sounds right, but consistency is important.)